custom-transformer-2.0_2.11/src/main/scala/org/apache/spark/ml/StringMerge.scala

```diff
@@ -0,0 +1,45 @@
+package org.apache.spark.ml
```

Comment:
PopularWordsEstimator needs be inside org.apache.spark.ml( package) because it is using few Private class (say private[ml] object DefaultParamsWriter) which has scope to this package. Others I will move to other package.

Reply:
StringMerge moved to other package